src/containers/catalog/CatalogComponent.js

```diff
@@ -1,12 +1,15 @@
 import React, { PropTypes } from 'react';
-import { EntitySetList } from './components/EntitySetList';
-import AuthService from '../../utils/AuthService';
+import { connect } from 'react-redux';
+import { EntitySetList } from '../../components/entityset/EntitySetList';
+import { createEntitySetListRequest } from './CatalogActionFactories';
+import AsyncContent from '../../components/asynccontent/AsyncContent';
 
-
-export class CatalogComponent extends React.Component {
+class CatalogComponent extends React.Component {
   static propTypes = {
-    auth: PropTypes.instanceOf(AuthService)
-  }
+    asyncState: PropTypes.object.isRequired,
+    entitySets: PropTypes.array.isRequired,
```

Comment:
use of `PropTypes.array` is discouraged. the recommended way is `React.PropTypes.arrayOf(React.PropTypes.shape({...}))`